docs/http_transport.md

```diff
@@ -0,0 +1,79 @@
+How to use SDK HTTP Transport
+========================================================
+
+The SDK concept is based on using HTTP transport mechanism to send requests.
```

Comment:
Consider: "The SDK functionality relies on an "HTTP Transport" mechanism to handle HTTP communication (requests / responses). The mechanism is decoupled from the SDK implementation itself and allows for usage of different libraries that can handle the HTTP communication."

Reply:
fixed